### PR TITLE
Olm

### PR DESCRIPTION
**Pull request by Jiri-Kremser**
manifests for [OLM](https://github.com/operator-framework/operator-lifecycle-manager)
the manifest for CRD approach has currently troubles because of the issue #162  (workaround is to use explicit list of namespaces to be watched, instead of `*`)

### CODE REVIEW

File: manifest/olm/crd/sparkclusteroperator.0.3.3.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,110 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: sparkoperator.v0.3.3
+  namespace: placeholder
+  annotations:
+    categories: apache spark, apache, spark
+    certified: false
```
Comment (SamiSousa) [inline manifest/olm/crd/sparkclusteroperator.0.3.3.clusterserviceversion.yaml]: Should be the string `"false"`

Comment (Jiri-Kremser) [reply manifest/olm/crd/sparkclusteroperator.0.3.3.clusterserviceversion.yaml]: ty :+1:

Comment (SamiSousa) [reply manifest/olm/crd/sparkclusteroperator.0.3.3.clusterserviceversion.yaml]: The change is needed to work with Operator Hub

Comment (SamiSousa) [reply manifest/olm/crd/sparkclusteroperator.0.3.3.clusterserviceversion.yaml]: The change is needed to work with Operator Hub

**Comment by SamiSousa** [comment]
Also the CSV needs an `installModes` field to be installed into OLM from Operator Hub (the following link is to a file in a PR): https://github.com/operator-framework/operator-lifecycle-manager/blob/bfb21629baaec3807d04247f7ea67efa252879fa/pkg/api/apis/operators/v1alpha1/clusterserviceversion_types.go#L26

**Pull request merged**